BLSQ/iaso
# POLIO-1194 correct process for mail vaccine auth cron jobs

The cronjobs were not linked to a task. This PR fix it.

Related JIRA tickets : POLIO-1194

## Self proofreading checklist


## Changes

The function are now a task launched by a cron.

## How to test

Quite hard to test on a local setup as it's directly linked Elastic Beanstalk
Check the code, the tests or run the function directly from a python shell.
